packages/components/psammead-grid/src/index.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { Grid } from '.';
+import { ExampleParagraph, ExampleFigure } from './helpers';
+
+describe('Grid component', () => {
+  shouldMatchSnapshot(
+    'should render Grid wrapper with Grid',
+    <Grid
+      columns={{ group3: 6, group4: 8, group5: 20 }}
+      startOffset={{ group3: 1, group4: 1, group5: 5 }}
+      enableGelMargins
+      enableGelGutters
+      enableGelMaxWidths
+      wrapper
```

Comment:
I do like the naming of the `leaf` prop. However, since there are fewer wrapper nodes than leaf nodes, I think this current setup will be easier for users of the Grid component.  Happy to consider changing the name of `wrapper`, if you have another more clearer suggestion.

Reply:
Thats fair, SGTM :)

Reply:
@dr3 I've updated implementation to follow your suggested layout. This is due to the feedback that it makes more sense to have a grid and a grid item. So I've called the wrapper `<Grid>` and then the grid item is `<Grid item>` - so requires a boolean prop called 'item' to be added to it.